mozci/push.py

```diff
@@ -17,6 +18,7 @@
 )
 from mozci.util.taskcluster import find_task_id
 
+HGMO_AUTOMATION_RELEVANCE_URL = "https://hg.mozilla.org/integration/{branch}/json-automationrelevance/{rev}"  # noqa
```

Comment:
Unrelated to this patch, but I just noticed that the `Push` class won't work on `central` or `try` because we hardcode `integration` into the URL. I'll file an issue to fix this.

Reply:
#53.